### ISSUE TITLE
Committed changes not visible - RBE shows old code

### ISSUE THREAD

**Issue by psyklopp** [rust-lang/rust-by-example#1879]
I have added a TLDR; section at the end. Apologies if this is a very simple error but I just wanted to report this. I am still a beginner.

## Issue 

I added the following snippet of code - https://github.com/rust-lang/rust-by-example/pull/1878 and the commits were merged. 

```
    /* Compound types - Array and Tuple */

    // Array signature consists of Type T and length as [T; length].
    let my_array: [i32; 5] = [1, 2, 3, 4, 5];

    // Tuple is a collection of values of different types 
    // and is constructed using parentheses ().
    let my_tuple = (5u32, 1u8, true, -5.04f32);
```

But the changes are not visible in the code editor on the relevant page. We can check it here: [2. Primitives](https://doc.rust-lang.org/rust-by-example/primitives.html)

I also fixed the workflow status on README.md but that is visible.

## Similar Issue

**May 16** 

- https://github.com/rust-lang/rust-by-example/issues/1846, the author mentions that the following comment is incorrect and should be changed. ```3.1 - Custom Types > Structures```

```
// Incorrect comment

// `bottom_right.y` will be the same as `point.y` because we used that field
// from `point`
```
to
```
// Correct comment

// `bottom_right.y` will be the same as `another_point.y` because we used that field
// from `another_point`
```

**June 6** 

- In the pull request https://github.com/rust-lang/rust-by-example/pull/1858 , the above was corrected. We can see the corrected comment in the repository - https://github.com/rust-lang/rust-by-example/blob/master/src/custom_types/structs.md

But the same is not visible when we view the page online - [3.1 Structures](https://doc.rust-lang.org/rust-by-example/custom_types/structs.html)

**July 22**

- The issue https://github.com/rust-lang/rust-by-example/issues/1866 also occurs because, though the changes were done in the `structs.md` file, they are not visible correctly on the website.

## TLDR;

The .md file and the corresponding generated .html are different.

Check for line 60, 61 in [3.1 Structures .md file](https://github.com/rust-lang/rust-by-example/blob/master/src/custom_types/structs.md) 

They are not shown correctly on the website: https://doc.rust-lang.org/rust-by-example/custom_types/structs.html

Same for the page [Primitives](https://doc.rust-lang.org/rust-by-example/primitives.html) - [primitives.md file](https://github.com/rust-lang/rust-by-example/blob/master/src/primitives.md)

**Comment by psyklopp** [comment]
Just a note: Generating RBE locally after `mdbook build` and `mdbook serve`, everything is visible correctly. The above issues do not appear.
